.github/workflows/unit-test.yaml

```diff
@@ -0,0 +1,27 @@
+name: unit tests
```

Comment:
Can we add a separate target for linting similar to what @indradhanush did with https://github.com/platform9/barista/pull/56?